src/repositories/CardRepository.ts

```diff
@@ -0,0 +1,10 @@
+import { Repository } from '../core/Repository';
+
+class CardRepository extends Repository {
+  constructor() {
+    super('card-list');
+  }
+}
+
+export default new CardRepository();
+
```

Comment:
클래스로 extends 된 인스턴스를 관리하는 팁입니다.

생성자 함수에 인자를 명시하여 어떤 역할을 하는 파라미터인지 타인에게 인지시키고, 해당 값을 super로 올려주게 하면 직관성이 높아지며 외부에서 무엇을 할 지 선택사항이 많아집니다.

```suggestion
class CardRepository extends Repository {
  constructor(storeKey:string) {
    super(storeKey);
  }
}

export default new CardRepository('card-list');

```